<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <column>
       <property name="text">
        <string>ProjectView</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QTextEdit" name="textEdit_3"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget_2">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>標準出力</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QTextEdit" name="textEdit">
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;stdout&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>標準エラー</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QTextEdit" name="textEdit_2">
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;stderr&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_C"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_R"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>編集(&amp;E)</string>
    </property>
    <addaction name="action_U"/>
    <addaction name="action_R_2"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C_2"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_D"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_F"/>
    <addaction name="separator"/>
    <addaction name="action_N_2"/>
    <addaction name="action_B"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>ビルド(&amp;B)</string>
    </property>
    <addaction name="action_C_3"/>
    <addaction name="action_A_2"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>実行(&amp;R)</string>
    </property>
    <addaction name="action_S_2"/>
    <addaction name="action_A_3"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>設定(&amp;O)</string>
    </property>
    <addaction name="action_S_3"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="actionCPUExSolver_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
   <addaction name="menu_R"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="text">
    <string>プロジェクトの新規作成(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>プロジェクトを開く(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>ファイルを閉じる(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>ファイルを保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>すべて保存(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>最近使ったファイル(&amp;R)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>終了(&amp;X)</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>もとに戻す(&amp;U)</string>
   </property>
  </action>
  <action name="action_R_2">
   <property name="text">
    <string>やりなおし(&amp;R)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>切り取り(&amp;T)</string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>コピー(&amp;C)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>貼り付け(&amp;P)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>削除(&amp;D)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>すべて選択(&amp;A)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>検索・置換(&amp;F)</string>
   </property>
  </action>
  <action name="action_N_2">
   <property name="text">
    <string>次のタブへ移動(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="action_B">
   <property name="text">
    <string>前のタブへ移動(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="action_C_3">
   <property name="text">
    <string>ビルド(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>すべてビルド(&amp;A)</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="text">
    <string>実行(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>すべて実行(&amp;A)</string>
   </property>
  </action>
  <action name="action_S_3">
   <property name="text">
    <string>設定(&amp;S)</string>
   </property>
  </action>
  <action name="actionCPUExSolver_A">
   <property name="text">
    <string>CPUExSolverについて(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_X</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_O</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_C</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_S</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>showProjectViewContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_U</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_R_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditRedo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_T</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditCut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_C_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_P</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditPaste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_D</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_A</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditSelectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_F</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditFind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_N_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditNextTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_B</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textEditBackTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCPUExSolver_A</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_C_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>build()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_S_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_A_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>buildAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_A_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>runAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createProject()</slot>
  <slot>newProject()</slot>
  <slot>saveFile()</slot>
  <slot>openFile(QTreeWidgetItem*,int)</slot>
  <slot>closeFile()</slot>
  <slot>showProjectViewContextMenu(QPoint)</slot>
  <slot>textEditUndo()</slot>
  <slot>textEditRedo()</slot>
  <slot>textEditCut()</slot>
  <slot>textEditCopy()</slot>
  <slot>textEditPaste()</slot>
  <slot>textEditDelete()</slot>
  <slot>textEditSelectAll()</slot>
  <slot>textEditFind()</slot>
  <slot>textEditNextTab()</slot>
  <slot>textEditBackTab()</slot>
  <slot>showAbout()</slot>
  <slot>run()</slot>
  <slot>runAll()</slot>
  <slot>build()</slot>
  <slot>buildAll()</slot>
 </slots>
</ui>
